runtime/jcl/common/mgmtmemory.c

```diff
@@ -76,25 +76,77 @@ Java_com_ibm_java_lang_management_internal_MemoryMXBeanImpl_getNonHeapMemoryUsag
 	omrthread_monitor_enter(javaVM->classTableMutex);
 	classLoader = javaVM->internalVMFunctions->allClassLoadersStartDo(&walkState, javaVM, 0);
 	while (NULL != classLoader) {
-		UDATA *udataFreeListBlock = classLoader->ramClassUDATABlockFreeList;
-		J9RAMClassFreeListBlock *tinyFreeListBlock = classLoader->ramClassTinyBlockFreeList;
-		J9RAMClassFreeListBlock *smallFreeListBlock = classLoader->ramClassSmallBlockFreeList;
-		J9RAMClassFreeListBlock *largeFreeListBlock = classLoader->ramClassLargeBlockFreeList;
-		while (NULL != udataFreeListBlock) {
-			used -= sizeof(UDATA);
-			udataFreeListBlock = *(UDATA **) udataFreeListBlock;
+		J9RAMClassUDATABlockFreeList *udataFreeListBlock = &classLoader->ramClassUDATABlocks;
```

Comment:
I suggest making the names of these local variables more similar to the fields they are being assigned to for clarity. For example this one can be `ramClassUDATABlocksPtr`.